deep_qa/data/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .datasets.dataset import Dataset, IndexedDataset, TextDataset
-
+from .datasets.dataset import Dataset
```

Comment:
Thanks.  Though at this point I'm mostly just looking for API-level feedback - I can find all of these issues with tests and pylint later, I don't want to waste your time doing this manually.